Makefile

```diff
@@ -1,4 +1,4 @@
-__VERSION__ = "0.5.1"
+__VERSION__ = "0.5.2"
```

Comment:
Version bump should be handled in release flow. Multiple PR:s could be merged before releasing a new version.